src/Databases/PostgreSQL/DatabaseMaterializedPostgreSQL.cpp

```diff
@@ -155,6 +194,59 @@ StoragePtr DatabaseMaterializedPostgreSQL::tryGetTable(const String & name, Cont
 }
 
 
+String DatabaseMaterializedPostgreSQL::getTablesList(const String & except) const
```

Comment:
Unclear method interface. Method return string, not list. What is `except` -- it's also comma-separated string?